app/src/main/java/com/zeapo/pwdstore/crypto/PgpActivity.kt

```diff
@@ -316,7 +316,8 @@ class PgpActivity : AppCompatActivity(), OpenPgpServiceConnection.OnBound {
                                             entry.totpSecret,
                                             Date().time / (1000 * entry.totpPeriod),
                                             entry.totpAlgorithm,
-                                            entry.digits)
+                                            entry.digits
+                                    )
```

Comment:
looks like an accidental newline

Reply:
I just don't know which is the right way...